services/dynamic-scheduler/Dockerfile

```diff
@@ -146,7 +146,7 @@ HEALTHCHECK --interval=30s \
   --timeout=20s \
   --start-period=30s \
   --retries=3 \
-  CMD ["python3", "services/dynamic-scheduler/docker/healthcheck.py", "http://localhost:8000/"]
+  CMD ["python3", "services/dynamic-scheduler/docker/healthcheck.py", "http://localhost:8000/health"]
```

Comment:
why is this one different than the other services?

Reply:
I really thought it was OK since it's present in other services.

![Screenshot 2024-11-26 at 11 12 19](https://github.com/user-attachments/assets/e8d99620-7256-4e6a-9999-4e7da07fe518)